### DISCUSSION TITLE
Quorum node for a 2-node cluster

### DISCUSSION THREAD

**Discussion by martinzwirner** [dotnet/dotNext#202] [Q&A] [answered]
Hi,

I want to use DotNext.Net.Cluster to implement high availability in a cluster of two machines. I know that a third node is required to achieve majority votes. This third node must never become a leader though, because the service only runs on the first two machines.

Can I configure the third node in this way, so that it votes, but never becomes a leader?

Thank you!

**Comment by sakno** [comment, marked as answer]
Yes, you can. This kind of node is called `Standby` node. It receives replicas from the leader node, can vote for new leader, but it will never become a leader. Check across [this](https://dotnet.github.io/dotNext/features/cluster/raft.html) documentation.
